<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVTKWidget" name="qvtkWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menu_Filters">
    <property name="title">
     <string>&amp;Filters</string>
    </property>
    <addaction name="action_detect_plane"/>
   </widget>
   <widget class="QMenu" name="menumode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_Selection"/>
    <addaction name="action_Annotation"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="action_BoundBox"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Filters"/>
   <addaction name="menumode"/>
   <addaction name="menu_Tool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_BoundBox"/>
   <addaction name="separator"/>
   <addaction name="action_detect_plane"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="CLoudOperation_dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>120</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>128</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Detect Ground</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DistanceThreshold:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="distanceThreshold_lineEdit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="removePlane_checkBox">
        <property name="text">
         <string>remove plane</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyPlaneDetection_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_detect_plane">
   <property name="text">
    <string>&amp;DetectPlane</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>&amp;Visiualize</string>
   </property>
  </action>
  <action name="action_Selection">
   <property name="text">
    <string>&amp;Selection</string>
   </property>
  </action>
  <action name="action_Annotation">
   <property name="text">
    <string>&amp;Annotation</string>
   </property>
  </action>
  <action name="action_BoundBox">
   <property name="text">
    <string>&amp;BoundBox</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
